app/assets/javascripts/modules/project.js

```diff
@@ -70,4 +70,92 @@ angular.module('probedock.project', [ 'probedock.api', 'probedock.forms', 'probe
     }
   })
 
-;
+.directive('projectHealthChart', function() {
+  return {
+    restrict: 'E',
+    controller: 'ProjectHealthChartCtrl',
+    templateUrl: '/templates/project-health.html',
+    scope: {
+      project: '='
+    }
+  };
+})
+
+
+.controller('ProjectHealthChartCtrl', function(api, $scope) {
+  $scope.chart = {
+    data: [],
+    labels: ['passed', 'failed', 'inactive'],
+    colors: ['#62c462', '#ee5f5b', '#fbb450'],
+    params: {}
+
+  };
+
+  $scope.projectVersionChoices = [];
+
+  $scope.$watch('project', function(value) {
+    if (value) {
+      fetchMetrics();
+      fetchProjectVersionChoices();
+    }
+  });
+
+  var ignoreChartParams = true;
+  $scope.$watch('chart.params', function(value) {
+    if (value && !ignoreChartParams) {
+      fetchMetrics();
+    }
+
+    ignoreChartParams = false;
+  }, true);
+
+  function fetchProjectVersionChoices() {
+    api({
+      url: '/projectVersions',
+      params: {
+        projectId: $scope.project.id
+      }
+    }).then(function(res) {
+      $scope.projectVersionChoices = res.data;
+    });
+  }
+
+  function fetchMetrics() {
+    var params;
+
+    if ($scope.chart.params.projectVersion) {
+      params = {
+        projectVersionId: $scope.chart.params.projectVersion.id
+      };
+    }
+    else {
+      params = {
+        projectId: $scope.project.id
+      };
+    }
+
+    return api({
+      url: '/metrics/projectHealth',
+      params: params
+    }).then(showMetrics);
+  }
+
+  function showMetrics(response) {
+    if (!response.data) {
+      return;
+    }
+
+    if (!$scope.chart.rawData) {
```

Comment:
I find the use of this `$scope.chart.rawData` variable not very intuitive. I had to read this and the template several times to understand what it was for. Maybe it would be better to store it as `$scope.projectHealth` or `$scope.chart.projectHealth` which would make it clearer that it's the result of the API call. What do you think?